core/domain/state_domain.py

```diff
@@ -46,17 +46,24 @@ class AnswerGroup(python_utils.OBJECT):
     """
 
     def __init__(
-            self, outcome, rule_inputs, rule_input_translations_mapping,
-            training_data, tagged_skill_misconception_id):
+            self, outcome, rule_types_to_inputs,
+            rule_types_to_inputs_translations, training_data,
+            tagged_skill_misconception_id):
         """Initializes a AnswerGroup domain object.
 
         Args:
             outcome: Outcome. The outcome corresponding to the answer group.
-            rule_inputs: dict. A dictionary mapping rule type to a list of rule
-                inputs.
-            rule_input_translations_mapping: dict. A dictionary mapping
-                rule_type to a dictionary that maps abbreviated language code
-                to the translated rule inputs.
+            rule_types_to_inputs: dict. A dictionary mapping rule type (str) to
```

Comment:
For this and the next arg, I suggest illustrating with an example.

Reply:
Done